### ISSUE TITLE
About the Preprocessing of input images.

### ISSUE THREAD

**Issue by cslxiao** [wkentaro/pytorch-fcn#41]
This project used different preprocessing method with the Pytorch pretrained models. The pretrained models use RGB values normalized to [0, 1]. While in pytorch-fcn, images are BGR values and not normalized to [0, 1], only center normalized. So how did it works?
 
See this on pytorch discuss:
All pretrained torchvision models have the same preprocessing, which is to normalize using the following mean/std values: https://github.com/pytorch/examples/blob/master/imagenet/main.py#L92-L93164 (input is RGB format)
[ref: https://discuss.pytorch.org/t/how-to-preprocess-input-for-pre-trained-networks/683/2]

**Comment by wkentaro** [comment]
I use VGG16 model pretrained using caffe, and distributed at Caffe-Zoo. And that's why I use different preprocessing.
This is common, and the original author of FCN also used that model.
